src/components/TextField/TextField.tsx

```diff
@@ -0,0 +1,40 @@
+interface Props {
+  title: string;
+  name: string;
+  value?: string;
+  label: string;
+  placeholder: string;
+  hasError?: boolean;
+  required?: boolean;
+  onChange: (newValue: string) => void;
+}
+
+export const TextField: React.FC<Props> = ({
+  title,
+  name,
+  value = title,
+  label,
+  placeholder,
+  hasError = false,
+  required = true,
+  onChange,
+}) => {
+  return (
+    <div className="field">
+      <label className="label" htmlFor="title">
```

Comment:
The 'htmlFor' attribute in the label should match the 'id' of the corresponding input element